website/docs/d/organization.html.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: "github"
+page_title: "GitHub: github_organization"
+description: |-
+  Get an organization.
+---
+
+# github_organization
+
+Use this data source to retrieve basic information about a GitHub Organization.
+
+## Example Usage
+
+```hcl
+data "github_organization" "test" { name = "github" }
+```
+
+## Attributes Reference
+
+ * `plan` - The plan name for the organization account
```

Comment:
Nitpick: though just used for testing, we could still add this data-source to the list of data-sources in `github.erb` as this page is in place.

Reply:
added to `github.erb` in a recent commit